reference/conanfile/tools/meson/mesondeps.rst

```diff
@@ -0,0 +1,104 @@
+.. _MesonDeps:
+
+MesonDeps
+=========
+
+.. warning::
+
+    These tools are still **experimental** (so subject to breaking changes) but with very stable syntax.
+    We encourage the usage of it to be prepared for Conan 2.0.
+
+:ref:`MesonToolchain<conan-meson-toolchain>` normally works together with :ref:`PkgConfigDeps<PkgConfigDeps>` to manage all the dependencies,
+but sometimes we need to gather some flags coming from ``Autotools`` tool so that's what ``MesonDeps`` is meant for.
```

Comment:
Maybe add explicitly the ``find`` that Meson could be doing internally, saying that this is something not typical but that sometimes happen?